src/Validator/Rules/ProvidedRequiredArgumentsOnDirectives.php

```diff
@@ -66,13 +66,15 @@ static function (FieldArgument $arg) : string {
             }
 
             $requiredArgsMap[$def->name->value] = Utils::keyMap(
-                $arguments ? array_filter($arguments, static function (Node $argument) : bool {
-                    return $argument instanceof NonNullTypeNode &&
-                        (
-                            ! isset($argument->defaultValue) ||
-                            $argument->defaultValue === null
-                        );
-                }) : [],
+                $arguments
+                    ? array_filter($arguments, static function (Node $argument) : bool {
+                        return $argument instanceof NonNullTypeNode &&
+                            (
+                                ! isset($argument->defaultValue) ||
+                                $argument->defaultValue === null
+                            );
+                    })
```

Comment:
Are you sure this is supposed to be on the same level as the `?` and `:` and not 3 spaces to the right so it's below the **a**`rray_filter`?

I might have seen a few other places too but before I nitpick on all of them… ;)

Reply:
This is the PHPStorm default style, and it is applied consistently.